src/SFML/Window/GlContext.cpp

```diff
@@ -73,9 +68,38 @@
 
 #endif
 
+#if !defined(GL_MULTISAMPLE)
```

Comment:
GLLoader.hpp takes care of the OpenGL functions and definitions (the "rendering stuff"). GL_MULTISAMPLE and all the stuff underneath are related to context management and/or window/pixel format specifications. They aren't supposed to be defined together. This is the way it is in the official headers from Khronos as well. Previously they were in glext.h (not GL.h), but I got rid of that because we don't need a 10000 line file just for a few definitions :wink:.